cmake/tribits/ci_support/CDashQueryAnalyzeReport.py

```diff
@@ -62,7 +62,7 @@
 # 'YYYY-MM-DD' value.
```

Comment:
Have the changes in the file been tested?

Reply:
Yes, extensively in:

* https://github.com/TriBITSPub/TriBITS/blob/master/test/ci_support/CDashQueryAnalyzeReport_UnitTests.py

I will put these tests up against anything being written in 1400 (but not as overkill as I used to write 10 years ago after reading "Working Effectively with Legacy Code").

As I mentioned yesterday, the TriBITS snapshot into Trilinos does not contain any automated tests.  Those are kept in the external TriBITS repo where all TriBITS development is done.